core/client/scenes/scene-world.js

```diff
@@ -71,8 +64,11 @@ WorldScene = new Phaser.Class({
 
     // Notes: tilesets with extrusion are required to avoid potential black lines between tiles (see https://github.com/sporadic-labs/tile-extruder)
     this.input.on('wheel', (pointer, gameObjects, deltaX, deltaY) => {
-      const zoom = Math.min(Math.max(this.cameras.main.zoom + (deltaY / zoomConfig.delta), zoomConfig.min), zoomConfig.max);
-      this.cameras.main.setZoom(zoom);
+      const clamp = (value, min, max) => Math.min(Math.max(value, min), max);
```

Comment:
We could move this declaration in the `client/helpers.js` file because I think we will need it elsewhere. (By the way, I don't really know how garbage collection is managed with a function declaration like that in Javascript)